### PR TITLE
feat(tutorials): add code for svelte pagination tutorial

### PR DESCRIPTION
**Pull request by Tammibriggs**

### CODE REVIEW

File: tutorials/svelte-pagination/src/App.svelte [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<script>
+	import {Allbooks} from './data'
+	import Book from './components/Book.svelte';
+</script>
+
+<main>
+	<div class='books'>
+		{#each Allbooks as book}
```
Comment (endymion1818) [inline tutorials/svelte-pagination/src/App.svelte:8]: This formatting is a little strange. Did you mean to convert to tabs here? Also it's not consistent with other files.

**Review by endymion1818 — COMMENTED** [review]
Only one small comment to address, thanks.

File: tutorials/svelte-pagination/src/App.svelte (hunk 1 above)
Comment (Tammibriggs) [reply tutorials/svelte-pagination/src/App.svelte:8]: Sorry, let me fix that.

Comment (Tammibriggs) [reply tutorials/svelte-pagination/src/App.svelte:8]: For some reason it looks strange on GitHub. Buy on my local machine its fine. I wonder why

Comment (endymion1818) [reply tutorials/svelte-pagination/src/App.svelte:8]: Someone else had this issue recently, I think it's due to some setting on VS Code. Please invest some time in fixing it Tammi.

**Review by endymion1818 — APPROVED** [review]

**Pull request merged**